CHANGELOG.md

```diff
@@ -2,14 +2,16 @@
 
 ## Master (Unreleased)
 
+## 2.5.0 (2021-09-21)
+
 * Declare autocorrect as unsafe for `ExpectChange`. ([@francois-ferrandis][])
 * Fix each example for `RSpec/HookArgument`. ([@lokhi][])
 * Exclude unrelated Rails directories from `RSpec/DescribeClass`. ([@MothOnMars][])
```

Comment:
Should we change `VersionChanged` for `DescribeClass` from 2.4.1 to 2.5.0 in `config/default.yml`?